_docs/use_functions.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Functions
+category: Common Concepts
+---
+
+You've already encountered the `main` function, the entrypoint of any Glu program, as well as the `func` keyword which is needed to declare a function.
+
+Procedural languages, like Glu, structure programs as a series of functions (also known as procedures) that call one another.
+This is why functions are indispensable in Glu programming.
+
+```glu
+func main() {
+    print("Hello World!");
+}
+```
+
+## Declare a Function
+
+To declare a function, three key components are required:
+- The `func` keyword
+- The function name
+- A list of any necessary parameters
+
+The body of the function is enclosed in curly brackets `{}`, which indicate to the compiler where the function begins and ends.
+
+In the example below, the function prototype of `my_func` specifies the function name and _parameters_: `param1` and `param2`, both of which are of type `Int`.
+
+![Glu Function Declaration Explanation](/assets/img/function_explaining.png)
```

Comment:
Add better alt text, the article cannot be understood by a blind person otherwise, as the image isn't just an illustration, it is part of your explanation